app/controllers/application_controller.rb

```diff
@@ -1,11 +1,13 @@
 class ApplicationController < ActionController::API
-
   include ActionController::HttpAuthentication::Token::ControllerMethods
-
   def authenticate
     authenticate_or_request_with_http_token do |token|
       @auth_user = User.find_by(token: token)
       @auth_user != nil ? true : false
+      @auth_user != nil ? true : false
+      @auth_user != nil ? true : false
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/RedundantConditional: This conditional expression can just be replaced by @auth_user != nil.